README.md

```diff
@@ -10,13 +10,24 @@ Jekyll Faker is a Jekyll/Liquid wrapper around the Faker gem, it allows you to d
 
 ## Installing
 
+Add one of the following lines to your site's `Gemfile`, and then run `bundle install` on the command-line.
+
 ```ruby
 gem "jekyll-faker"
 gem "jekyll-faker", {
   git: "https://github.com/anomaly/jekyll-faker.git"
 }
 ```
 
+Add `jekyll-faker` to the `plugins:` section in your site's `_config.yml` file.
+
+```yml
+plugins:
+  - jekyll-faker
+```
```

Comment:
I'm more a fan of recommending people use bundler, over the "Github Pages" way, would you mind removing this in favor of just adding `group: :jekyll_plugins` to the other directions?  Once it's in that group, it no longer needs to be listed inside of `plugins` (which is something that Jekyll is very bad at elaborating to users... and often leads to confusion.)

Reply:
I'm sorry, I'm confused by what you're asking for here. You want me to remove the reference to the plugin from the config file and somewhere put a `group: :jekyll_plugins`? That looks like Ruby code so I suspect it's not in the `_config.yml` file. Are you saying make a group in the `Gemfile` and list the plugin under that group, and then somehow bundler knows to add it to Jekyll's config?

Reply:
It belongs in the gem file not the `_config.yml`, so yes.